xgboost/notebooks/Online Prediction with XGBoost.ipynb

```diff
@@ -0,0 +1,627 @@
+    "\n",
+    "# get one person that makes <=50K and one that makes >50K to test our model.\n",
+    "print('Show a person that makes <=50K:')\n",
+    "print('\\tFeatures: {0} --> Label: {1}\\n'.format(data[0], test_labels[0]))\n",
+    "\n",
+    "with open('less_than_50K.json', 'w') as outfile:\n",
+    "  json.dump(data[0], outfile)\n",
+    "\n",
+    "  \n",
+    "print('Show a person that makes >50K:')\n",
+    "print('\\tFeatures: {0} --> Label: {1}'.format(data[2], test_labels[2]))\n",
+    "\n",
+    "with open('more_than_50K.json', 'w') as outfile:\n",
+    "  json.dump(data[2], outfile)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use Gcloud to make online predictions\n",
+    "Use the two people (as seen in the table) gathered in the previous step for the gcloud predictions.\n",
+    "\n",
+    "| **Person** | age | workclass | fnlwgt | education | education-num | marital-status | occupation |\n",
+    "|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:\n",
+    "| **1** | 25| 4 | 226802 | 1 | 7 | 4 | 7 |\n",
+    "| **2** | 28| 2 | 336951 | 7 | 12 | 2 | 11 |\n",
+    "\n",
+    "| **Person** | relationship | race | sex | capital-gain | capital-loss | hours-per-week | native-country || (Label) income-level|\n",
+    "|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:||:-:\n",
+    "| **1** | 3 | 2 | 1 | 0 | 0 | 40 | 38 || False (<=50K) |\n",
+    "| **2** | 0 | 4 | 1 | 0 | 0 | 40 | 38 || True (>50K) |\n",
+    "\n",
+    "\n",
+    "Creating a model version can take several minutes, check the status of your model version to see if it is available."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME  DEPLOYMENT_URI                        STATE\r\n",
+      "v1    gs://xgboost-tutorial-notebook-test/  CREATING\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! gcloud ml-engine versions list --model $MODEL_NAME"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME  DEPLOYMENT_URI                        STATE\r\n",
+      "v1    gs://xgboost-tutorial-notebook-test/  READY\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! gcloud ml-engine versions list --model $MODEL_NAME"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Test the model with an online prediction using the data of a person who makes <=50K.\n",
+    "\n",
+    "Note: If you see an error, the model from Part 4 may not be created yet as it takes several minutes for a new model version to be created."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[0.09139516949653625]\r\n"
```

Comment:
Can you add clarification on what the output means? (This one is supposed to be close to zero, since it was someone who makes <50K, right?)

Or 

Should we have it match the Label we showed in our Table? True/False?